doc changes (#5)
OpenZeppelin/openzeppelin-sui-marketplace: README.md

End-to-end example of a small on-chain market on **Sui**: items are priced in **USD cents** (stablecoin-style), while buyers can pay in **multiple currencies** using **oracle prices**.

This repo is a pnpm workspace containing:
- a Move package `packages/dapp/move/oracle-market`,
- a CLI/script layer for localnet + seeding + owner/buyer flows `dapp/scripts`
- state artefact captured in (`packages/dapp/deployments`) 
- a Next.js UI `packages/ui`,
- a docs site with learning path to help transition from EVM/Solidity to Sui/Move `packages/learn`.
- a tooling layer with integration test harness `packages/tooling`

More detail (workspace layering rules, folder layout): `docs/01-repo-layout.md`.

## Prerequisites
- Node.js 22+ [Install](https://nodejs.org/en/download)
- pnpm [Install](https://pnpm.io/installation)
- Sui CLI [Install](https://docs.sui.io/guides/developer/getting-started/sui-install)

## Environment Setup

> Set up your environment so you have active address and localnet running so you can publish and interact with the oracle-market package

Reviewer: ```suggestion
Set up your environment with an active address and a running localnet to publish and interact with the `oracle-market` package.
```
